packages/core/src/stryker-cli.ts

```diff
@@ -181,6 +181,7 @@ export class StrykerCli {
         `Choose whether or not to clean the temp dir (which is "${defaultOptions.tempDirName}" inside the current working directory by default) after a successful run. The temp dir will never be removed when the run failed for some reason (for debugging purposes).`,
         parseBoolean
       )
+      .option('--enableConsoleEmojis', 'Choose whether or not emoji will be shown in CLI output. Default option is false.')
```

Comment:
Since we're calling it an experimental feature, let's not put it as a cli argument for now.